oxd-common/src/main/java/org/gluu/oxd/common/response/GetOpDiscoveryConfigResponse.java

```diff
@@ -0,0 +1,564 @@
+package org.gluu.oxd.common.response;
+
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.annotate.JsonProperty;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author yuriyz
+ */
+public class GetOpDiscoveryConfigResponse implements IOpResponse {
```

Comment:
add `@JsonIgnoreProperties(ignoreUnknown = true)`. We don't want to fail if OP adds/removes/change something.

Reply:
Changes done.